core/src/test/java/com/sinch/sdk/core/TestHelpers.java

```diff
@@ -1,12 +1,20 @@
 package com.sinch.sdk.core;
 
+import com.sinch.sdk.core.models.OptionalValue;
+import java.util.Comparator;
 import org.assertj.core.api.Assertions;
 import org.assertj.core.api.recursive.comparison.RecursiveComparisonConfiguration;
 
 public class TestHelpers {
 
-  private static final RecursiveComparisonConfiguration recursiveComparisonConfiguration =
-      RecursiveComparisonConfiguration.builder().withStrictTypeChecking(true).build();
+  static Comparator<OptionalValue> optionalValueComparator = (a, b) -> a.equals(b) ? 0 : 1;
+  private static RecursiveComparisonConfiguration recursiveComparisonConfiguration =
+      RecursiveComparisonConfiguration.builder()
+          // .
```

Comment:
There is some cleanup to do here

Reply:
![image](https://github.com/user-attachments/assets/57260e0a-dcec-44db-a1d0-b3b8ba19a696)

Done.